apps/web/app/globals.css

```
@@ -2,15 +2,56 @@
+  --color-app-border-strong: rgba(255, 255, 255, 0.14);
+  --color-app-text: rgba(255, 255, 255, 0.95);
+  --color-app-text-muted: rgba(255, 255, 255, 0.55);
+  --color-app-text-faint: rgba(255, 255, 255, 0.35);
+
+  --color-violet: #6b4bff;
+  --color-violet-bright: #a67cff;
+  --color-violet-deep: #4a2fd6;
+  --color-success-neon: #4dffa8;
+  --color-danger-glow: #ff4d6d;
+
   --font-sans: Poppins, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.9.0)

[error] Expected "Poppins" to be "poppins" (value-keyword-case)

(value-keyword-case)